src/frontend/src/binder/expr/function.rs

```diff
@@ -520,6 +520,63 @@ impl Binder {
                         ))))
                     }
                 ))),
+                ("pg_table_size", guard_by_len(1, raw(|binder, inputs|{
+                        let input = &inputs[0];
+                        let bound_query = binder.bind_get_object_size_select("pg_table_size", input)?;
+                        Ok(ExprImpl::Subquery(Box::new(Subquery::new(
+                            BoundQuery {
+                                body: BoundSetExpr::Select(Box::new(bound_query)),
+                                order: vec![],
+                                limit: None,
+                                offset: None,
+                                with_ties: false,
+                                extra_order_exprs: vec![],
+                            },
+                            SubqueryKind::Scalar,
+                        ))))
+                    }
+                ))),
+                ("pg_relation_size",raw(|binder, inputs|{
+                        if inputs.len() >= 1 && inputs.len() <= 2{
+                            Err(ErrorCode::ExprError("pg_relation_size takes 1 or 2 arguments".into()))?
+                        }
+                        let input = &inputs[0];
+
+                        // This function, on Postgres, offers the ability to get the size of forks of a 
+                        // table in addition to the main fork. RW's HummockVersionStats does not provide
+                        // such data so this is currently implemented as an alias for `pg_table_size`.
+                        //
+                        // See: https://stackoverflow.com/a/41991566 and https://pgpedia.info/p/pg_relation_size.html
```

Comment:
Given the semantics of `pg_relation_size` does it make sense to implement this in RW? At best, it can be made an alias for `pg_table_size` but then would it be a valid implementation and would doing this cause issues for users?

Reply:
I prefer not supporting `pg_relation_size` with two arguments because RW doesn't have concepts like `fsm`, `vm`, `init` for a relation.

Reply:
One concern with not supporting `pg_relation_size(arg1, arg2)`, is that any existing tools built for PG would be at risk of breaking if run against RW. From the User's perspective this would be both confusing and frustrating.

This also makes me wonder: if we're not going to support the full form of `pg_relation_size` should we support it at all?  Even with just the 1 parameter version of `pg_relation_size` it's not returning the information that Postgres would return. Finally, if we're going to only support the 1 argument form of `pg_relation_size`, we're already willing to break some queries that work for Postgres, so why not just rip the bandaid off and not support `pg_relation_size` at all rather than have a partial implementation?

> If we do want to just support the 1 argument version, then I'll update the code to return an informative error to the user if they supply 2 arguments for the function.

Reply:
Agreed. Not supporting `pg_relation_size` for now also sounds reasonable to me.